src/main/java/picard/analysis/directed/HsMetricCollector.java

```diff
@@ -101,6 +98,21 @@ public HsMetrics convertMetric(final TargetMetrics targetMetrics) {
         return hsMetrics;
     }
 
+    /**
+     * Updates several of the metrics in the provided Metrics object so that they better reflect the particular target strategy
+     *
+     * @param hsMetrics the metrics object to fix
+     * @param bases the total number of bases
+     * @param selectedPairs the total number of selected pairs
+     * @param selectedUniquePairs the number of selected pairs that are unique
+     */
+    protected void updateMetrics(final HsMetrics hsMetrics, final long bases, final long selectedPairs, final long selectedUniquePairs) {
```

Comment:
This looks like it could be `static`, unless we expect users to override this method? If it can be static, I would also consider making it a `HsMetrics` member function, e.g. 
```
public void updateMetrics(final long bases, final long selectedPairs, final long selectedUniquePairs) {
    ...
```

Reply:
Yeah, that's the whole point.  I'm overriding it (in a picard-private class that extends this one).